op-e2e/e2eutils/disputegame/output_game_helper.go

```diff
@@ -450,6 +450,12 @@ func (g *OutputGameHelper) waitForNewClaim(ctx context.Context, checkPoint int64
 
 func (g *OutputGameHelper) AttackWithTransactOpts(ctx context.Context, claimIdx int64, claim common.Hash, opts *bind.TransactOpts) {
 	g.t.Logf("Attacking claim %v with value %v", claimIdx, claim)
+
+	claimData, err := g.game.ClaimData(&bind.CallOpts{Context: ctx}, big.NewInt(claimIdx))
+	g.require.NoError(err, "Failed to get claim data")
+	pos := types.NewPositionFromGIndex(claimData.Position)
+	g.fillBondTransactOpts(ctx, pos.Attack().ToGIndex(), opts)
```

Comment:
Should we reset the `Value` in opts again after we modify it or take a copy to modify? I'm guessing it's fine now but is the kind of thing that can be a nasty surprise.

Reply:
Good note. It's better to copy and modify here.